### PR TITLE
Updating FunctionalData, Watcher, QSC, LaTeX, InfDE for 0.5

### PR DESCRIPTION
**Pull request by rened**

### CODE REVIEW

File: FunctionalData/versions/0.1.1/requires [hunk 1]
```diff
@@ -0,0 +1 @@
+julia 0.4
```
Comment (tkelman) [inline FunctionalData/versions/0.1.1/requires]: you're using Compat but not listing it in REQUIRE

Comment (rened) [inline FunctionalData/versions/0.1.1/requires]: Thanks, fixed by removing the dependency on Compat.

**Comment by rened** [comment]
Squashed the commits, hope thats ok / preferred.

**Comment by tkelman** [comment]
I'd generally prefer fewer packages per PR, easier to review and merge individually that way.

File: FunctionalData/versions/0.1.2/sha1 [hunk 2]
```diff
@@ -0,0 +1 @@
+a5fc40efe6a1f9f8983a2067b6395b6f0b01cc8d
```
Comment (tkelman) [inline FunctionalData/versions/0.1.2/sha1]: push the git tag so it shows up at https://github.com/rened/FunctionalData.jl/releases

Comment (rened) [inline FunctionalData/versions/0.1.2/sha1]: done

**Comment by rened** [comment]
I also prefer one-by-package. Thought the expectation here was rather to have fewer PRs, merging things.

Will gladly move back to individual PRs ;-)  Thanks for reviewing!

**Comment by tkelman** [comment]
These tags appear to have broken KShiftsClustering.jl on Julia 0.4: http://pkg.julialang.org/logs/KShiftsClustering_0.4.log

**Pull request merged**